mw-webapp/src/component/image/Image.tsx

```diff
@@ -1,11 +1,28 @@
 import clsx from "clsx";
+import {Cy, Modal} from "src/component/modal/Modal";
 import styles from "src/component/image/Image.module.scss";
 
+/**
+ * Data attributes for cypress testing
+ */
+interface CyDataImage extends Cy {
+
+  /**
+   * Data attribute for cypress testing
+   */
+  onZoom?: string;
```

Comment:
it's not onZoom cy, it's cy for image even if it's not zoomed
Rename to imageCy or similar